docs/extending-documentation.md

```diff
@@ -0,0 +1,289 @@
+```
+
+and create a `babel.config.js` file in the root of your project:
+
+```js
+const path = require('path');
+const providenceExtendConfig = require('./providence-extend-docs-data.json');
+
+const extendDocsConfig = {
+  rootPath: path.resolve('.'),
+  changes: providenceExtendConfig,
+};
+
+module.exports = {
+  overrides: [
+    {
+      test: ['./node_modules/@lion/*/README.md', './node_modules/@lion/*/docs/*.md'],
+      plugins: [['babel-plugin-extend-docs', extendDocsConfig]],
+    },
+  ],
+};
+```
+
+As you can see, we import the providence output file and pass it to the plugin options as the `changes` property.
+
+Now, the babel plugin will run for the files that we specify in `test` property, and replace the imports properly, as well as the tag names inside JavaScript code snippets!
+
+> For the JavaScript code snippets, it only transforms the ones that use [MDJS](https://open-wc.org/mdjs/) syntax, e.g. \`\`\`js script, \`\`\`js story and \`\`\`js preview-story
+
+To ensure babel actually runs for your Storybook, you will have to add it to your es-dev-server configuration:
+
+`.storybook/main.js`:
+
+```js
+module.exports = {
+  stories: ['../node_modules/@lion/tabs/README.md', '../packages/**/!(*.override)*.md'],
+  esDevServer: {
+    nodeResolve: true,
+    watch: true,
+    open: true,
+    babel: true,
+  },
+};
+```
+
+You should now see the `LionTabs` instances transformed into your own `LeaTabs`!
+
+## Adjust documentation content
+
+In some cases you don't want to show all examples of how to use a component. Sometimes information is `Lion` specific, or perhaps in your design system you are not allowed to use a certain feature that we documented in `Lion`.
+
+In our example, we will show you have to remove the `Rationale` section that you would normally inherit from the `Lion` documentation.
+
+For this step we make use of a remark plugin for the MD content, similar to how you would use a babel plugin for JS content. It is called [Remark extend](https://lion-web-components.netlify.app/?path=/docs/tools-remark-extend--page).
+It will let you add, remove or replace sections or specific words.
+
+First of all we need to add the plugin to the `.storybook/main.js`:
+
+```js
+const fs = require('fs');
+const { remarkExtend } = require('remark-extend');
+
+function isLion(filePath) {
+  return filePath.indexOf('@lion/') !== -1;
+}
+
+function getLocalOverridePath(filePath, root = process.cwd()) {
+  const rel = filePath.substring(filePath.indexOf('/@lion/') + 7, filePath.length - 3);
+  return `${root}/packages/${rel}.override.md`;
+}
+
+module.exports = {
+  [...],
+  setupMdjsPlugins: (plugins, filePath) => {
+    if (!isLion(filePath)) {
+      return plugins;
+    }
+    const newPlugins = [...plugins];
+    const markdownIndex = newPlugins.findIndex(plugin => plugin.name === 'markdown');
+    const overridePaths = [`${process.cwd()}/.storybook/all.override.md`];
+    overridePaths.push(getLocalOverridePath(filePath));
```

Comment:
```suggestion
    const overridePaths = [`${process.cwd()}/.storybook/all.override.md`, getLocalOverridePath(filePath)];
```
probably nicer, now that we don't have a conditional installation override file.